### PR TITLE
Reinstate python tests missing from cmake lists

### LINKED ISSUE #586: Tests not included in test runner
Several tests are not included in CMakeLists, including 5 pytests:

tests/res/enk/test_runpath_list_ert.py 
tests/res/enkf/test_update.py
tests/res/enkf/data/test_custom_kw_config.py
tests/res/enkf/data/test_field_config.py
tests/res/util/test_ui_return.py

### PR DESCRIPTION
**Pull request by oyvindeide**
**Issue**
Resolves #586

### CODE REVIEW

**Comment by ghost** [comment]
Can one of the admins verify this patch?

**Comment by markusdregi** [comment]
Jenkins; add to whitelist

**Comment by oyvindeide** [comment]
There are a bunch of tests in libres which are importing ert_gui, will remove these tests

**Comment by oyvindeide** [comment]
retest this

**Comment by oyvindeide** [comment]
retest this please

File: python/tests/res/analysis/test_std_enkf_debug.py [hunk 1]
```diff
@@ -14,22 +14,31 @@
 #  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
 #  for more details.
 import sys
+import os
 
 import res
+
```
Comment (ManInFez) [inline python/tests/res/analysis/test_std_enkf_debug.py]: white space?

File: python/tests/res/analysis/test_std_enkf_debug.py [hunk 2]
```diff
@@ -14,22 +14,31 @@
...
+    return None
 
 class StdEnKFDebugTest(ResTest):
 
     def setUp(self):
         self.rng = RandomNumberGenerator(RngAlgTypeEnum.MZRAN, RngInitModeEnum.INIT_DEFAULT)
         if sys.platform.lower() == 'darwin':
-            self.libname = res.res_lib_path + "/std_enkf_debug.dylib"
+            self.libname = find_file(self.SOURCE_ROOT, "std_enkf_debug.dylib")
+
```
Comment (ManInFez) [inline python/tests/res/analysis/test_std_enkf_debug.py]: Whitespace?

**Review by ManInFez — APPROVED** [review]
LGTM :D

File: python/tests/res/analysis/test_std_enkf_debug.py (hunk 1 above)
Comment (oyvindeide) [reply python/tests/res/analysis/test_std_enkf_debug.py]: intentionally, from pep:
>Imports should be grouped in the following order:
>
>Standard library imports.
>Related third party imports.
>Local application/library specific imports.
>You should put a blank line between each group of imports.

Comment (oyvindeide) [reply python/tests/res/analysis/test_std_enkf_debug.py]: was intentional, but the library isn't actually used, so I'll remove the import

**Pull request merged**